static/app/views/replays/detail/issueList.tsx

```diff
@@ -0,0 +1,203 @@
+import {Fragment, useCallback, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+import keyBy from 'lodash/keyBy';
+
+import EventOrGroupExtraDetails from 'sentry/components/eventOrGroupExtraDetails';
+import EventOrGroupHeader from 'sentry/components/eventOrGroupHeader';
+import {PanelTable} from 'sentry/components/panels';
+import Placeholder from 'sentry/components/placeholder';
+import {DEFAULT_STREAM_GROUP_STATS_PERIOD} from 'sentry/components/stream/group';
+import GroupChart from 'sentry/components/stream/groupChart';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Group, NewQuery, PageFilters} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import theme from 'sentry/utils/theme';
+import useApi from 'sentry/utils/useApi';
+import {useLocation} from 'sentry/utils/useLocation';
+import useMedia from 'sentry/utils/useMedia';
+import useOrganization from 'sentry/utils/useOrganization';
+import withPageFilters from 'sentry/utils/withPageFilters';
+
+type Props = {
+  projectId: string;
+  replayId: string;
+  selection: PageFilters;
+};
+const columns = [t('Issue'), t('Graph'), t('Events'), t('Users')];
+
+function IssueList(props: Props) {
+  const organization = useOrganization();
+  const location = useLocation();
+  const api = useApi();
+  const isScreenLarge = useMedia(`(min-width: ${theme.breakpoints[2]})`);
+
+  const [issuesById, setIssuesById] = useState<Record<string, Group>>({});
+  const [issueStatsById, setIssuesStatsById] = useState<Record<string, Group>>({});
+
+  const getEventView = () => {
+    const {selection} = props;
+    const eventQueryParams: NewQuery = {
+      id: '',
+      name: '',
+      version: 2,
+      fields: ['count(issue)', 'issue'],
+      environment: selection.environments,
+      projects: selection.projects,
+      query: `replayId:${props.replayId} AND event.type:error`,
+    };
+    const result = EventView.fromNewQueryWithLocation(eventQueryParams, location);
+    return result;
+  };
+
+  const fetchIssueData = useCallback(async () => {
+    let issues;
+    try {
+      issues = await api.requestPromise(`/organizations/${organization.slug}/issues/`, {
+        includeAllArgs: true,
+        query: {
+          collapse: 'stats',
+          expand: ['owner', 'inbox'],
+          project: props.projectId,
+          query: `replayId:${props.replayId}`,
+        },
+      });
+
+      setIssuesById(keyBy(issues[0], 'id'));
```

Comment:
What happens if there are no issues? (I suppose it gets caught)

Reply:
If there's no errors coming from the discovery query we get the 'no data' result on the table. I've been operating on the assumption that if we don't pull any errors from the discovery query then we probably aren't going to see issues.

Reply:
The view is predicated upon pulling issues back. If the API call fails you'll just see the table headers, I can adjust that but I wasn't sure what course of action to take around that.